test/autorest/generated/lrogroup/pollers.go

```diff
@@ -101,7 +72,15 @@ func httpPollerPollUntilDone(ctx context.Context, p HTTPPoller, frequency time.D
 			time.Sleep(frequency)
 		}
 	}
-	return p.FinalResponse(ctx)
+	return p.FinalResponse(), nil
+}
+
+func (p *httpPoller) handleResponse(resp *azcore.Response) (*HTTPResponse, error) {
```

Comment:
It looks like this isn't actually used for things that return an `*http.Response`, can be omitted.

Reply:
You're right will remove

Reply:
Done